db/version_edit.cc

```diff
@@ -315,6 +324,7 @@ const char* VersionEdit::DecodeNewFile4From(Slice* input) {
         return "new-file4 custom field";
       }
       if (custom_tag == kTerminate) {
+        assert(f.min_timestamp.size() == f.max_timestamp.size());
```

Comment:
In case of error, we need to return a string literal to indicate error. THe assertion will be optimized away in opt build mode.

Reply:
ok, I added some return msg after assert.